orchagent/routeorch.cpp

```diff
@@ -1187,3 +1325,80 @@ bool RouteOrch::removeRoute(sai_object_id_t vrf_id, const IpPrefix &ipPrefix)
 
     return true;
 }
+
+bool RouteOrch::createRemoteVtep(sai_object_id_t vrf_id, const NextHopKey &nextHop)
+{
+    SWSS_LOG_ENTER();
+    EvpnNvoOrch* evpn_orch = gDirectory.get<EvpnNvoOrch*>();
+    VxlanTunnelOrch* tunnel_orch = gDirectory.get<VxlanTunnelOrch*>();
+    bool status = false;
+    int ip_refcnt = -2;
+
+    SWSS_LOG_NOTICE("Routeorch Add Remote VTEP %s, VNI %d, VR_ID %lx",
+            nextHop.ip_address.to_string().c_str(), nextHop.vni, vrf_id);
+    status = tunnel_orch->addTunnelUser(nextHop.ip_address.to_string(), nextHop.vni, 0, TNL_SRC_IP, vrf_id);
```

Comment:
Where is `TNL_SRC_IP` defined?

Reply:
It is defined in PR 1264.
https://github.com/Azure/sonic-swss/pull/1264/files : orchagent/vxlanorch.h